### DISCUSSION TITLE
Plugins

### DISCUSSION THREAD

**Discussion by Turkey2552** [Pryaxis/TShock#2221] [Q&A]
I just got this im trying to add a admin command plugin the zadmincommands and i put it in the folder but i dont know how to acess it please help

**Comment by Kojirremer** [comment]
I don't quite understand what your issue is. You just put the plugin's DLL into the ServerPlugins folder **before** turning the server on, and if it's a valid plugin it should start without issues, create its own config file, if needed, and then be available for use.
If it crashes at startup, whatever needs to be done depends on the crash itself. The plugin may be missing a dependency, or be outdated.
